tests/benchdnn/inputs/graph/pattern/harness_f32_all

```diff
@@ -134,6 +134,15 @@
 --reset --in-shapes=0:384x4096+1:4096x1024+2:384x1 --case=pattern/f32/matmul_post_ops_swish_fusion.json
 --reset --in-shapes=0:384x4096+1:4096x1024+2:1024 --case=pattern/f32/matmul_post_ops_swish_fusion.json
 --reset --case=pattern/f32/matmul_select.json
+# matmul broadcasting tests - 1D x nD with post mul fusion
+# 1D x 1D matmul, mul with scalar [1]
+--reset --in-shapes=0:128+1:128+3:1 --case=pattern/f32/matmul_1dx2d_post_mul_fusion.json
+# 1D x 1D matmul, mul with 2D broadcast [1,1]
+--reset --in-shapes=0:256+1:256+3:1x1 --case=pattern/f32/matmul_1dx2d_post_mul_fusion.json
+# 1D x 2D matmul, mul with 1D same shape [256]
+--reset --in-shapes=0:512+1:512x256+3:256 --case=pattern/f32/matmul_1dx2d_post_mul_fusion.json
+# 1D x 2D matmul, mul with 2D broadcast [1,512]
+--reset --in-shapes=0:1024+1:1024x512+3:1x512 --case=pattern/f32/matmul_1dx2d_post_mul_fusion.json
```

Comment:
If possible, please also extend the single matmul test case cover different broadcasting rules.